front-end/Components/MatchDetailScreen.js

```diff
@@ -0,0 +1,78 @@
+//install MapView from react-native-maps
+//https://youtu.be/su5v_Gg9Sxk?si=GDSgHHFhrUeqdIif
+import React, { useState, useEffect } from "react";
+import MapView, { Marker } from "react-native-maps";
+import { StyleSheet, Text, View, Button } from "react-native";
+import { venues } from "../data/venues";
+
+export default function MapScreen() {
+  const initialVenue = venues.find((venue) => venue.venue_id === 1);
+
+  const [mapRegion, setMapRegion] = useState({
+    latitude: initialVenue.venue_latitude,
+    longitude: initialVenue.venue_longitude,
+    latitudeDelta: 0.0922,
+    longitudeDelta: 0.0421,
+  });
+
+  const [selectedVenue, setSelectedVenue] = useState(initialVenue);
```

Comment:
I think this might benefit from being broken into smaller components - I think the map justifies being a component by itself (as shown in the component tree diagram).